nautobot/core/utils/lookup.py

```diff
@@ -76,7 +76,9 @@ def get_route_for_model(model, action, api=False):
 
     suffix = "" if not api else "-api"
     prefix = f"{model._meta.app_label}{suffix}:{model._meta.model_name}"
-    sep = "_" if not api else "-"
+    sep = ""
+    if action != "":
+        sep = "_" if not api else "-"
```

Comment:
Many of our routes right now do not end in `_detail` or `_retrieve`. For now, this works and we can choose to clean up our routes whenever instead of scope-creaping this.